### PR TITLE
terms: fix format and typos in privacy policy and cookie policy

### PR DESCRIPTION
**Pull request by lilin90**
- Fix and update format
- Fix Chinese punctuation
- Fix typos: Agolia > Algolia
- Make word consistent: pseudonymise > pseudonymize

@YiniXu9506 @dcalvin PTAL

### CODE REVIEW

File: markdown-pages/terms/privacy-policy.md [hunk 1]
```diff
@@ -52,28 +52,28 @@ We collect several data within product telemetry data as below :
...
+Our Sites and related online services use cookies and similar technologies to enable certain functionality and help collect information about your visit. "Cookies" are small text files, typically containing a unique string of letters and numbers, stored on your hard drive by a Site. When you return to the Site using the same browser, the Site can read the cookie and thereby gather information about your usage over time. Among other things, we use cookies and other technology to see which areas and features are popular and to count visits, which helps us to improve our Site, our Products, and your experience.
 
 The cookies we use include:
 - Web beacons
 - Performance cookies
 -Functionality cookies
```
Comment (YiniXu9506) [inline markdown-pages/terms/privacy-policy.md:72]: The list formate need to fix

File: markdown-pages/terms/privacy-policy.md [hunk 2]
```diff
@@ -52,28 +52,28 @@ We collect several data within product telemetry data as below :
...
 
 **Information collected through social media and other platforms.\*\*** We receive information about you when you engage with us through various third-party platforms, for example, by joining our Slack community or Google group, liking us on Facebook, connecting on LinkedIn, GitHub, or Meetup, following us on Twitter or Instagram, registering for training through WooCommerce, or sharing content from our Site on Facebook, Twitter, or LinkedIn. The data we exchange with these third-party platforms may depend upon your privacy settings with these platforms. You should review and consider adjusting your privacy settings on third-party websites and services before engaging. Do not provide us with any sensitive personal data through these platforms. We are not responsible for the data protection and use practices of these third-party platforms. Please see their privacy statements to learn how they use your information.
```
Comment (YiniXu9506) [inline markdown-pages/terms/privacy-policy.md:57]: The extra `**` need to be removed

File: markdown-pages/terms/privacy-policy.md (hunk 1 above)
Comment (lilin90) [reply markdown-pages/terms/privacy-policy.md]: Fixed.

File: markdown-pages/terms/privacy-policy.md (hunk 2 above)
Comment (lilin90) [reply markdown-pages/terms/privacy-policy.md]: Removed. @dcalvin FYI.

**Review by YiniXu9506 — APPROVED** [review]
LGTM

**Review by dcalvin — APPROVED** [review]
LGTM

**Pull request merged**